packages/crayons-core/src/components/country-phone/config.js

```diff
@@ -0,0 +1,9 @@
+import countries from './countries.json';
+
+export const data = countries;
```

Comment:
Can you kindly retain this name as countries.

Reply:
Fixed : ✅